conf/ocsci/submariner_upstream.yaml

```diff
@@ -0,0 +1,9 @@
+ENV_DATA:
+  submariner_source: "upstream"
+  submariner_url: "https://get.submariner.io"
+  submariner_info_file: "broker-info.subm"
+  submariner_default_gateway_node: "compute-1"
```

Comment:
in case of ipi we will not have `compute-1` we need to set this value based on node name

Reply:
yes we need only one node. Can be any any node from worker node

Reply:
done, now fetching the first node in the workers list